code/campaignclassic/src/main/java/com/adobe/marketing/mobile/BasicTemplateNotificationBuilder.java

```diff
@@ -73,19 +76,76 @@ class BasicTemplateNotificationBuilder {
         expandedLayout.setTextViewText(
                 R.id.notification_body_expanded, pushTemplate.getExpandedBodyText());
 
-        // set any custom colors if needed
-        AEPPushNotificationBuilder.setCustomNotificationColors(
+        return createNotificationBuilder(
```

Comment:
With the final cleanup after this PR, I am assuming that this will be split in to two methods and changed to accept a template or an intent to reduce the number of parameters being sent here.